src/ralph/lib/mixins/models.py

```diff
@@ -102,3 +102,15 @@ class TaggableMixin(models.Model):
 
     class Meta:
         abstract = True
+
+
+class PreviousStateMixin(models.Model):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        fields = self._meta.get_all_field_names()
```

Comment:
https://docs.djangoproject.com/en/1.9/ref/models/meta/#migrating-from-the-old-api